src/view/ViewCommandHandlers.js

```diff
@@ -266,6 +266,24 @@ define(function (require, exports, module) {
         }
     }
 
+
+    /**
+     * Font smoothing setter to set the anti-aliasing type for the code area on Mac.
+     * @param {string} aaType The antialiasing type to be set.It can take either "subpixel-antialiased" or "antialiased"
+     */
+    function setMacFontSmoothingType(aaType) {
+        var $editor_holder  = $("#editor-holder"),
+            hasClass        = $editor_holder.hasClass("subpixel-aa");
```

Comment:
We don't really need this `hasClass`. `add/removeClass` will no-op in case the class is not present.